lms/djangoapps/progress/management/commands/recalculate_progress_for_users.py

```diff
@@ -0,0 +1,103 @@
+"""
+One-time data migration script -- shoulen't need to run it again
+"""
+import logging
+from optparse import make_option
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.db.models import Q
+
+from progress.models import StudentProgress, CourseModuleCompletion
+from progress.signals import is_valid_progress_module
+from student.models import CourseEnrollment
+from xmodule.modulestore.django import modulestore
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Recalculate progress entries for the specified course(s) and/or user(s)
+    """
+
+    def handle(self, *args, **options):
+        option_list = BaseCommand.option_list + (  # pylint: disable=unused-variable
```

Comment:
These options are not working, because the `option_list` attribute is supposed to be declared on the class, not inside the `handle` method. When you fix that, you can remove the pylint warning.

You probably copied this from the other command in the `progress` app, which also gets this wrong.